src/js/components/Navigation/HeaderBar.jsx

```diff
@@ -168,11 +168,13 @@ export default class HeaderBar extends Component {
   }
 
   toggleAboutMenu () {
-    this.setState(prevState => ({ aboutMenuOpen: !prevState.aboutMenuOpen }));
```

Comment:
Setting state using the prevState wasn't working, switched it to using destructuring instead. This combined with the fix in components/Navigation/HeaderBarAboutMenu.jsx seems to have resolved it.